src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -16,8 +16,10 @@
 using Nethermind.Core.Specs;
```

Comment:
general question: did we address somehow this https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7702.md#transaction-propagation?

Reply:
No, the suggested mitigations has not been implemented. However it should be fairly straight forward to restrict tx pool to one targeting tx per delegated EOA.